lizrd/support/code_versioning.py

```diff
@@ -59,6 +59,7 @@ def version_code(self):
             self.stash_if_necessary()
             self.revert_status = 1
             self.repo.git.checkout(b=self.name_for_branch)
+            self.unstash_if_necessary()
```

Comment:
The code lacked applying stashed changes to the branch that is to be pushed to neptune along with the experiment launch.